shell/platform/windows/public/flutter_windows.h

```diff
@@ -32,6 +32,9 @@ typedef struct FlutterDesktopView* FlutterDesktopViewRef;
 struct FlutterDesktopEngine;
 typedef struct FlutterDesktopEngine* FlutterDesktopEngineRef;
 
+// The unique identifier for a view.
+typedef int64_t FlutterDesktopViewId;
```

Comment:
This `FlutterDesktopViewId` typedef is new, the original design used `int64_t` directly.